examples/data-solution-extension/README.md

```diff
@@ -0,0 +1,30 @@
+# About
+
+This example is a 2 step approach to setup Data Solution Extension (DSE) from scratch and publish it
+to tenant for consumption. One **must** have Data Solution .iso file to use this example.
```

Comment:
```suggestion
This example is a 2-step approach to setup Data Solution Extension (DSE) from scratch and publish it
to a tenant for consumption. One **must** have Data Solution .iso file to use this example.
```
Shall we provide the URL to the .ISO file?

Reply:
Fixed.